languages/c/ast/ast_c_build.ml

```diff
@@ -627,7 +628,7 @@ and expr env e =
       A.Generic (tk, (l, (expr env e, args), r))
   | IdSpecial (SizeOf, tk) -> A.IdSpecial (SizeOf, tk)
   | ConstructedObject (_, _) ->
-      logger#error "BUG PARSING LOCAL DECL PROBABLY";
+      Logs.err (fun m -> m ~tags "BUG PARSING LOCAL DECL PROBABLY");
```

Comment:
This should be Logs.err, like you did, but then that means every semgrep users will now see those errors ...

I think that we should configure logging so that any Logs.err or Logs.info with a ~tags should be masked and shown only when you use --debug really.

Reply:
My conclusion, as mentioned on Slack, is that no log message should be seen on stdout or stderr by the command-line user unless they turn it on (or run a server loop). In the end, these error-level messages are debugging helps just like all the other levels.

Reply:
yes but then you should change this one to Logs.debug, because Logs.err will get displayed to the user I think.